package.json

```diff
@@ -17,18 +17,20 @@
     "start": "NODE_ENV=production node --trace-warnings ./src/server/server.babel.js",
     "start:tunnel": "export $(grep ^ULTRAHOOK_API_KEY .env | tr -d '\"') && ultrahook -k $ULTRAHOOK_API_KEY dev 3000",
     "dev": "if [ ! -f \"dll/vendors.json\" ] ; then npm run build:dll ; fi ; NODE_ENV=development node --trace-warnings ./src/server/server.babel.js",
-    "build": "rimraf build && npm run build:relay && concurrently \"npm run build:client\" \"npm run build:server\"",
+    "build": "rimraf build && npm run build:deps && concurrently \"npm run build:client\" \"npm run build:server\"",
     "build:deploy": "rimraf build && npm run build:server && npm run build:client-deploy",
     "build:min": "rimraf build && concurrently \"npm run build:client-min\" \"npm run build:server\"",
     "bs": "npm run build && npm start",
     "debug:stripe": "node ./src/server/billing/debug.babel.js",
     "quickstart": "npm run db:migrate && npm run build && npm run start",
     "build:client": "NODE_ENV=production webpack --config ./webpack/prod.babel.js",
+    "build:deps": "concurrently \"npm run build:theme\" \"npm run build:dll\" \"npm run build:relay\"",
```

Comment:
dll is only used for local development. why would we build it in prod?

Reply:
Was mostly a DX change. I can strike it.